matrix_sdk_crypto/src/machine.rs

```diff
@@ -731,14 +724,40 @@ impl OlmMachine {
     }
 
     /// Get a verification request object with the given flow id.
-    pub fn get_verification_request(&self, flow_id: &EventId) -> Option<VerificationRequest> {
+    pub fn get_verification_request(
+        &self,
+        flow_id: impl AsRef<str>,
+    ) -> Option<VerificationRequest> {
         self.verification_machine.get_request(flow_id)
     }
 
     async fn update_one_time_key_count(&self, key_count: &BTreeMap<DeviceKeyAlgorithm, UInt>) {
         self.account.update_uploaded_key_count(key_count).await;
     }
 
+    async fn handle_to_device_evnet(&self, event: &AnyToDeviceEvent) {
```

Comment:
```suggestion
    async fn handle_to_device_event(&self, event: &AnyToDeviceEvent) {
```

Reply:
Thanks, is this part of you typos PR or do I need to add this separately?

Reply:
It's in the typos PR